### DISCUSSION TITLE
Testing on kedro

### DISCUSSION THREAD

**Discussion by BryanDePrediqtdata** [kedro-org/kedro#1522] [Idea]
Greetings, do you have a example to how do a test to a pipeline?
i have a multiples pipelines without tests and i want to start to use dataops

**Comment by noklam** [comment]
@BryanDePrediqtdata Depends on what you want to test. `node` is a natural fit for unit-testing. If you need to test the entire pipeline, it's best to have some small data fixture so it runs quickly enough. To do that you could do `kedro run --env=test` and have your test data defined.

**Reply by BryanDePrediqtdata** [reply]
Do you have any example, no matter what test only i have a template to start, please.
